### PR TITLE
End register ECT journey if teacher has completed their induction

### PR DESCRIPTION
**Pull request by TobyRet**
### Ticket 

#687 

### Changes

- Update TestGuidanceComponent: add 'Notes' column so that additional information about a teacher can be included.
- If a teacher has completed their induction, the user is redirected to a new error page 

### Video

https://github.com/user-attachments/assets/0fd12031-c790-402f-8f00-db38843b1fcb

### CODE REVIEW

File: app/views/schools/register_ect/induction_completed.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<% page_data(title: "You cannot register #{@ect.full_name}", error: false) %>
+
+<p class="govuk-body">Our records show <%= @ect.full_name %> has already completed their induction and is not eligible for further ECT training.</p>
+
+<p class="govuk-body"><%= govuk_link_to("Register another ECT", schools_register_ect_find_ect_path) %></p>
```
Comment (claire-hughez) [inline app/views/schools/register_ect/induction_completed.html.erb:5]:
```suggestion
<p class="govuk-body">Our records show <%= @ect.full_name %> has already completed their induction and is not eligible for further training as an ECT.</p>

<p class="govuk-body"><%= govuk_link_to("Register another ECT", schools_register_ect_find_ect_path) %></p>
```

Comment (tonyheadford) [reply app/views/schools/register_ect/induction_completed.html.erb:5]: Could this be something like: "... has already completed their induction and therefore cannot be registered as an ECT." ?

**Review by tonyheadford — APPROVED** [review]
Looks fine from the dev perspective

File: app/views/schools/register_ect/induction_completed.html.erb (hunk 1 above)
Comment (claire-hughez) [reply app/views/schools/register_ect/induction_completed.html.erb:5]: That works for me! What do you think Rob?

**Pull request merged**